shared/logger/types.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+// Separate type file because standard is failing
+
+type Timestamp = number
+
+export type LogLevels = 'Error' | 'Warn' | 'Info' | 'Action' | 'Debug'
+
+export type LogLine = [Timestamp, string]
+export type LogLineWithLevel = [LogLevels, Timestamp, string]
+
+export type LogFn = (...s: Array<string>) => void
```

Comment:
Do we want to stringify everything at call time? Would be a shame to lose the behavior of console.log that lets you drill down

Reply:
Sure, we can change this type